site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -178,5 +178,43 @@ pipeline:
         tier: mysql
 ```
 
+### `selectors`
+
+Selectors can be used to target specific resources for function execution. Selectors 
```

Comment:
Don't call this "Resource selectors" or "selectors". It's too generic, has bad SEO and potentially ambiguous. I would call this Function Targets Selector consistently everywhere.

Reply:
We did consider naming it `targets`, but realized it wasn't consistent with the conventions in the kubernetes world where terms `selector` (in Deployment), or `nodeSelector` or `labelSelector` and `namespaceSelector` are used commonly to select resources.

`targetSelectors` was another term we considered, but then felt that 'selectors' within `pipeline/function` context should not feel ambiguous so decided to go with `selectors`. I think this is something we should get more feedback (may be in fishfood as well) @phanimarupaka  

With respect to docs, I agree, we should use consistent term such as `target selector` or `function target selector`. Definitely, `Function Target selector` is very specific.